windows/win_nssm.ps1

```diff
@@ -42,6 +42,33 @@ $dependencies = Get-Attr $params "dependencies" -default $null
 $user = Get-Attr $params "user" -default $null
 $password = Get-Attr $params "password" -default $null
 
+Function Nssm-IsOutputMisencoded
+{
+    #this assumes winrm is running, which I think is a safe assumption
+    #one future option would be to fetch any running service and ask nssm its status
+    $sampleOutput = invoke-expression "nssm status winrm"
+    return [bool]($sampleOutput -cnotlike "SERVICE_RUNNING")
+}
+
+#abstract the calling of nssm because some PowerShell environments
+#mishandle its stdout(which is unicode) as UTF8
+Function Nssm-Invoke
+{
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$cmd
+    )
+    $results = invoke-expression "nssm $cmd"
```

Comment:
A much easier way to fix this would be to temporarily reset Powershell's Console OutputEncoding to Unicode, then call nssm in a `Try` block, and restore the encoding in a `Finally` block. Let Powershell do the re-encoding- because of the way NSSM is setting its output to UTF8, something in the middle is turning it into UTF16. It should behave this way everywhere (unless you've somehow installed an ANSI-compiled version of nssm).